### ISSUE TITLE
Linear sustain broken on pAI's

### ISSUE THREAD

**Issue by Jamarkus127** [tgstation/tgstation#65320]
Reporting client version: 514.1578

## Round ID:
[179557](https://scrubby.melonmesa.com/round/179557)

## Reproduction:

no reproduction, though its doing the same thing as https://github.com/tgstation/tgstation/issues/64992

linear sustain dosnt want to function with pAI's either. exact same issue as described in the link above, just with pAI's

**Comment by jlsnow301** [comment]
Related #65008

**Comment by timothymtorres** [comment]
This should have been fixed by #65008.  If it didn't ping me and I will reopen this issue.

**Comment by Jamarkus127** [comment]
@timothymtorres yep. just tested it, was busy. still dosnt seem to work. I think pAI code is different.

**Comment by timothymtorres** [comment]
Duplicate of #64992
